### DISCUSSION TITLE
照着 nshen 的v2配置，配置完了自己的一份，分享一下，哈哈

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#104] [Show and tell]
https://github.com/HUAHUAI23/nvim64
<img width="867" alt="8" src="https://user-images.githubusercontent.com/43649186/188250040-9eb31793-c5d9-4f12-a738-ae5013b536ed.png">

**Comment by nshen** [comment]
酷啊

**Reply by HUAHUAI23** [reply]
哈哈哈，谢谢
